bower.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "hawkular-ui-services",
   "description": "WIP:Angular Services for the Hawkular Console.",
-  "version": "0.8.15",
+  "version": "0.8.16",
```

Comment:
No need to version. In fact it will mess things up because the git repo needs to be tagged with the same version.
All you need is the git sha to add to hawkular's bower.json.